mbed_app.json

```diff
@@ -1,22 +1,33 @@
 {
     "config": {
         "hostname": {
-            "help": "The demo will try to connect to this web address on port 80.",
+            "help": "The demo will try to connect to this web address on port 80 (or port 443 when using tls).",
             "value": "\"ifconfig.io\""
+        },
+        "use-tls-socket": {
+            "value": true
         }
     },
     "target_overrides": {
         "*": {
             "nsapi.default-wifi-security": "WPA_WPA2",
             "nsapi.default-wifi-ssid": "\"YOUR_SSID\"",
             "nsapi.default-wifi-password": "\"YOUR_PASSWORD\"",
-            "platform.stdio-baud-rate": 115200
+            "platform.stdio-baud-rate": 115200,
+            "mbed-trace.enable": false,
```

Comment:
Is mbed-trace configuration required for the example ?

Reply:
you probably want traces on when using tls - discussed in readme